foster55f/palette-picker-be
# Fix server error so app and server are correctly connected

### What's this PR do?
This fixes our bug we ran into with our server not running correctly when we ran nodemon server.js.  We were creating two different apps in our app/server files, instead of just importing the app we were creating into server.js.

### Where should the reviewer start?
Check out the files and the lines that were deleted.  You'll notice that I removed a bunch of lines from server.js because those were creating another, new app.

### How should this be manually tested?
Pull this branch down and run nodemon server.js.  Then go to the localhost URL and see the message displayed!
